### PR TITLE
5284 neuroscience index

### PR DESCRIPTION
**Pull request by ckline-tryptic**
Issue: [5284](https://github.com/openstax/cnx-recipes/issues/5284)
image

Title styling will be handled by [this card](https://github.com/openstax/cnx-recipes/issues/5275).

### CODE REVIEW

**Review by kswanson33 — APPROVED** [review]

**Review by kswanson33 — APPROVED** [review]

File: styles/books/neuroscience/book.scss [hunk 1]
```diff
@@ -131,6 +131,14 @@ $ChapterIntroType: side;
...
 @include use('TOC', 'carnival:::NoUnitToc');
 
+// EOB
+@include add_settings((
+  AppendixGlossary: (
+    _selectors: (".appendix > .column-container"),
+    'H3AppendixGlossaryTitle:::color': (_ref: "colorMap:::neuroscienceHeaderColor"),
+    'AppendixGlossaryContainer:::column-count': 2,
+  ),
+));
```
Comment (wilkus27) [inline styles/books/neuroscience/book.scss:141]: We are keeping all the settings together on the top and all `use` methods at the bottom. Would you mind to move this over `@include use('BookRoot', "common:::BookRoot");` (L: 128)?

File: CHANGELOG.md [hunk 2]
```diff
@@ -23,7 +24,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+* Style `tables` in `lifespan-development`
+* Style `lo` in `lifespan-development`
+* Style `chapter-outline` in `lifespan-development`
+* Style figures in `lifespan-development`
+* Style headers in `lifespan-development`
+* Create basic files and style TOC in `lifespan-development`
+
+## [v1.140.0] - 2023-11-17
+
+* Fix EOC bands in `nursing-internal`
```
Comment (wilkus27) [inline CHANGELOG.md:51]: I think conflicts were resolved in a wrong way. There are repetitions in a CHANGELOG. Would you mind to compare this list of logs with one from main branch to make sure that versions which were already tagged are not changed and that nothing is missing from `Unreleased` one?

**Review by wilkus27 — CHANGES_REQUESTED** [review]

File: styles/books/neuroscience/book.scss [hunk 3]
```diff
@@ -124,13 +124,30 @@ $ChapterIntroType: side;
...
```
Comment (kswanson33) [inline styles/books/neuroscience/book.scss:150]: looks like AppendixGlossary `add_settings` is duplicated

**Review by kswanson33 — CHANGES_REQUESTED** [review]

File: styles/books/neuroscience/book.scss (hunk 3 above)
Comment (ckline-tryptic) [reply styles/books/neuroscience/book.scss]: Ha, yeah Jen and I were both working on it i think there was some collision :)

**Review by kswanson33 — APPROVED** [review]

**Review by wilkus27 — APPROVED** [review]

**Pull request merged**